### PR TITLE
Implemented ValidatedBifunctor

### PR DESCRIPTION
**Pull request by overfullstack**
Implemented ValidatedBifunctor

### CODE REVIEW

File: arrow-core/src/main/kotlin/arrow/core/extensions/validated.kt [hunk 1]
```diff
@@ -40,6 +41,12 @@ interface ValidatedFunctor<E> : Functor<ValidatedPartialOf<E>> {
   override fun <A, B> Kind<ValidatedPartialOf<E>, A>.map(f: (A) -> B): Validated<E, B> = fix().map(f)
 }
 
+@extension
+interface ValidatedBifunctor : Bifunctor<ForValidated> {
```
Comment (aballano) [inline arrow-core/src/main/kotlin/arrow/core/extensions/validated.kt]: Nice addition, thank you! Would you mind also adding the tests for BifunctorLaws in `ValidatedTest`?

Comment (overfullstack) [reply arrow-core/src/main/kotlin/arrow/core/extensions/validated.kt]: Sure

Comment (overfullstack) [reply arrow-core/src/main/kotlin/arrow/core/extensions/validated.kt]: @aballano Added the test

**Comment by pakoito** [comment]
Green

**Pull request merged**